packages/nimble-components/src/table/specs/table-column-specs/table-column-formatted-text.md

```diff
@@ -161,21 +161,65 @@ Follows the [Column Type Philosophy](/packages/nimble-components/src/table/specs
+-   `decimal-digits` - when format is `decimal`, a number that controls how many digits are shown to the right of the decimal separator. Defaults to 2 unless `decimal-maximum-digits` is specified, in which case a number of digits between 0 and `decimal-maximum-digits` are shown. It is invalid to specify both `decimal-digits` and `decimal-maximum-digits`. Formats other than `decimal` ignore `decimal-digits`.
+-   `decimal-maximum-digits` - when format is `decimal`, a number that controls the maximum number of digits shown to the right of the decimal separator. This has the same effect as `decimal-digits` except that trailing zeros are omitted. Possible values are from 0 to 20. It is invalid to specify both `decimal-digits` and `decimal-maximum-digits`. Formats other than `decimal` ignore `decimal-maximum-digits`.
 
 This column will display a blank cell when `typeof` the value is not `"number"` (i.e. if the value is `null`, `undefined`, not present, or has a different runtime data type). Note that IEE 754 numbers like Infinity, NaN, and -0 are type `"number"` so will be displayed how each formatter converts them. This will preserve values like `"∞"` and `"NaN"`.
 
-This column will trigger `invalidColumnConfiguration` on the table's validity state and will include flags in the column's validity state if its configuration can't be translated to a valid `Intl.NumberFormat` object. To provide better developer feedback about what's wrong with the configuration, the column could expose a public method like `createNumberFormat()` which would use the column's configuration to construct the formatter but allow any exceptions to be thrown.
+This column will trigger `invalidColumnConfiguration` on the table's validity state and will include flags in the column's validity state if both `decimal-digits` and `decimal-maximum-digits` are specified, or its configuration can't be translated to a valid `Intl.NumberFormat` object. To provide better developer feedback about what's wrong with the configuration, the column could expose a public method like `createNumberFormat()` which would use the column's configuration to construct the formatter but allow any exceptions to be thrown.
+
+A unit for the column may be configured by providing a `nimble-unit-<name>` element as content (in addition to the column label). Unit elements represent families of related units, e.g. `nimble-unit-byte` represents bytes, KB, MB, etc. Values are converted from a source unit (e.g. bytes) to the largest related unit (e.g. KB, MB, etc.) that can represent that value with magnitude >= 1. The source data for the column is expected to be given in the base unit specified in the tag name, e.g. for `nimble-unit-byte`, a source value should be a number of bytes. Note that unit family elements are not rendered but serve to allow selective loading of data for only the needed units. The initial set of unit elements are:
```

Comment:
I'm not sure I understand what you mean by "Note that unit family elements are not rendered but serve to allow selective loading of data for only the needed units." Do you mean the custom element (e.g. `nimble-unit-<name>`) isn't a rendered entity? If so, perhaps this can be better said as "Unit elements serve as configuration components only, and have no visual representation on their own".?

Reply:
The two points I was trying to communicate were that (1) they have no visual, and (2) the motivation for using an element-based API is to avoid always loading all translations for all languages for all units. I've reworded the sentence to hopefully make that clearer.